infra/base/terraform/addons.tf

```diff
@@ -76,24 +59,6 @@ module "eks_blueprints_addons" {
   cluster_version   = module.eks.cluster_version
   oidc_provider_arn = module.eks.oidc_provider_arn
 
-  #---------------------------------------
-  # Amazon EKS Managed Add-ons
-  #---------------------------------------
-  eks_addons = {
```

Comment:
Moved to EKS core add-ons. Any EKS addons will now be managed through EKS module, Refer to eks.tf file for core addons